static/sass/_patterns_strips.scss

```diff
@@ -231,3 +231,19 @@
     }
   }
 }
+
+@mixin jaas-p-strip-blue {
+  .p-strip--blue {
+    @extend %vf-strip;
+    background-color: #00365c;
+
+    &.is-dark {
+      color: $color-x-light;
+
+      .link--light {
```

Comment:
I guess you could use `.p-link--inverted` instead

Reply:
This actually isn't needed - I'll remove it.